tools/wake/main.cpp

```diff
@@ -228,6 +227,19 @@ void inspect_database(const CommandLineOptions &clo, Database &db, const std::st
   describe(matching_jobs, get_describe_policy(clo), db);
 }
 
+void inspect_database(const CommandLineOptions &clo, Database &db) {
+  // tagdag and history are db inspection queries, but are very different from the
+  // rest of the queries which operate on the jobs table.
+  if (clo.tagdag) {
+    JAST json = create_tagdag(db, clo.tagdag);
+    std::cout << json << std::endl;
```

Comment:
I feel like we should just move the printing into `create_tagdag` since all the other functions do it that way